Unicode/makeutype.c

```diff
@@ -630,6 +683,168 @@ static void dumparabicdata(FILE *header) {
     fclose( data );
 }
 
+static void buildtables(FILE *data,long unsigned int *dt,int s,int m,char *t) {
```

Comment:
Could we get a brief description of what the function does and the meanings of the parameters?